tests/test_validator_integration.py

```
@@ -0,0 +1,1014 @@
+
+        # config_cli requires a file on disk
+        import tempfile
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".yaml", delete=False) as f:
+            yaml.dump(hybrid, f)
+            tmp_path = f.name
+
+        try:
+            result = load_and_validate_config(
+                tmp_path, full_or_lora="full", method="span", validate=True,
+            )
+            config_ok = result.report.is_valid
```

Ruff (0.15.1)

[error] Local variable `config_ok` is assigned to but never used

Remove assignment to unused variable `config_ok`

(F841)